selfdrive/car/hyundai/hyundaican.py

```diff
@@ -187,14 +198,17 @@ def create_acc_opt(packer):
   }
   commands.append(packer.make_can_msg("SCC13", 0, scc13_values))
 
-  fca12_values = {
-    "FCA_DrvSetState": 2,
-    "FCA_USM": 1, # AEB disabled
-  }
-  commands.append(packer.make_can_msg("FCA12", 0, fca12_values))
+  # For cars that do not broadcast FCA messages, do not send FCA messages to prevent TCS13|ACCEnable fault
+  if has_fca:
+    fca12_values = {
+      "FCA_DrvSetState": 2,
+      "FCA_USM": 1,  # AEB disabled
+    }
+    commands.append(packer.make_can_msg("FCA12", 0, fca12_values))
```

Comment:
Checked some routes, FCA11 and FCA12 existing don't necessarily equal. FCA11 can be on the bus, but not FCA12 often.